File: 구름 알고리즘 미션/기초 구현/과연 승자는/과연 승자는.js
// Run by Node.js
const readline = require('readline');

(async () => {
	let rl = readline.createInterface({ input: process.stdin });
	let input = [];

	for await (const line of rl) {
    input.push(line);
    if(input.length === 3) rl.close();
	}
	
  const N = +input[0];
  const Alice = input[1].split(" ").map(v => +v);
  const Bob = input[2].split(" ").map(v => +v);
  let [scoreA, scoreB] = [0, 0];

  for(let i = 0 ; i < N ; i++){
    if(Alice[i] === Bob[i]){
      scoreA++; 
      scoreB++;
    } else if(Alice[i] < Bob[i]) {
      if(Bob[i] - Alice[i] === 7){
        scoreA += 3;
        scoreB--;
      } else {
        scoreB += 2;
      }
    } else {
      if(Alice[i] - Bob[i] === 7){
        scoreA--;
        scoreB += 3;
      } else {
        scoreA += 2;
      }
    }
  }

  console.log(scoreA, scoreB);
	process.exit();
})();

File: 구름 알고리즘 미션/기초 구현/과연 승자는/과연 승자는.py
N = int(input())
Alice = list(map(int, input().split(" ")))
Bob = list(map(int, input().split(" ")))

scores = [0, 0]
for i in range(len(Alice)):
	if Alice[i] == Bob[i]:
		scores[0] += 1
		scores[1] += 1
	elif Alice[i] > Bob[i]:
		if Alice[i] - Bob[i] == 7:
			scores[0] -= 1
			scores[1] += 3
		else:
			scores[0] += 2
	else:
		if Bob[i] - Alice[i] == 7:
			scores[0] += 3
			scores[1] -= 1
		else:
			scores[1] += 2
			
print(scores[0], scores[1])	